kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,266 @@
+
+This article describes how to set up your Temporal Cloud and SDK metrics, and use them as data sources in Grafana.
+
+The process for setting up observability includes the following steps:
+
+1. Create or get your Prometheus endpoint for Temporal Cloud metrics and enable SDK metrics.
+   - For Temporal Cloud, [generate a Prometheus HTTP API endpoint](#temporal-cloud-metrics-setup) on Temporal Cloud using valid certificates.
+   - For SDKs, [expose a metrics endpoint](#sdk-metrics-setup) where Prometheus can scrape SDK metrics and [run Prometheus](#prometheus-configuration-for-sdk-metrics) on your host. The examples in this article describe running Prometheus on your local machine where you run your application code.
+2. Run Grafana and [set up data sources for Temporal Cloud and SDK metrics](#data-sources-configuration-for-temporal-cloud-and-sdk-metrics-in-grafana) in Grafana. The examples in this article describe running Grafana on your local host where you run your application code.
+3. [Create dashboards](#grafana-dashboards-setup) in Grafana to view Temporal Cloud metrics and SDK metrics.
+
+If you're following through with the examples provided here, ensure that you have the following:
+
+- Root CA certificates and end-entity certificates. See [Certificate requirements](/cloud/how-to-manage-certificates-in-temporal-cloud#certificate-requirements) for details.
+- Set up your connections to Temporal Cloud using an SDK of your choice, and have some Workflows running on Temporal Cloud. See [Connect to a Cluster](application-development/foundations#connect-to-a-cluster) for details.
+- Prometheus and Grafana installed.
+
+<!-- truncate -->
+
+## Temporal Cloud metrics setup
+
+Before you set up your Temporal Cloud metrics, ensure that you have the following:
+
+- [Global Admin privileges](/cloud#account-level-roles) to the Temporal Cloud account.
+- [CA certificate and key](/cloud/how-to-manage-certificates-in-temporal-cloud) for the Observability integration. You will need the certificate to set up the Observability endpoint in Temporal Cloud.
+
+The following steps describe how to set up Observability on Temporal Cloud to generate an endpoint:
+
+1. Log in to Temporal Cloud UI as a Global Admin.
+2. Go to **Settings** and select **Integrations**.
+3. Select **Configure Observability** (if you’re setting it up for the first time) or click **Edit** in the Observability section (if it was already configured before).
+4. Add your root CA certificate (.pem) and save it.
+   Note that if an observability endpoint is already set up, you can append your root CA certificate here to use the generated observability endpoint with your instance of Grafana.
+5. To test your endpoint, run the following command on your host:
+   `curl -v --cert <path to your client-cert.pem> --key <path to your client-cert.key> "<your generated Temporal Cloud prometheus_endpoint>/api/v1/query?query=temporal_cloud_v0_state_transition_count"`.
```

Comment:
```suggestion
   ```
   curl -v --cert <path to your client-cert.pem> --key <path to your client-cert.key> "<your generated Temporal Cloud prometheus_endpoint>/api/v1/query?query=temporal_cloud_v0_state_transition_count"
```
```